src/Azure.Functions.Cli/Helpers/WorkerRuntimeLanguageHelper.cs

```diff
@@ -86,6 +107,22 @@ public static WorkerRuntime NormalizeWorkerRuntime(string workerRuntime)
             }
         }
 
+        public static string NormalizeLanguage(string languageString)
+        {
+            if (string.IsNullOrWhiteSpace(languageString))
+            {
+                return null;
```

Comment:
any callers actually expecting and handling a null? I wonder if this should also be an exception

Reply:
[This guy](https://github.com/Azure/azure-functions-core-tools/pull/1070/files#diff-26a1bb4213c73dc23de6444d3ffc7ab3R133) expects it to be null right now. Just so, it can get the default for that worker. But I can change to catch the exception and then do the default if you prefer that?

Reply:
I think you are right, because that codepath should never really be hit. And something must have gone wrong if it did. So, probably throwing an exception is better. I have updated the PR.